examples/05_collections/06_filter_products_in_collection/query.graphql

```diff
@@ -0,0 +1,25 @@
+# You can use the Storefront API to filter products in a collection using product filters.
+
+# This functionality lets you build a desired customer experience on a storefront, such as the ability to narrow down the search results that you display to customers.
+
+# Products in collections can be filtered by type, vendor, variant options, price, stock and metafield value.
+
+# Please note there are requirements to using product filters in collections here - https://shopify.dev/custom-storefronts/products-collections/filter-products#requirements
+
+# In the following example, products in the collection that have the "shoes" product type are returned.
+
+# Further examples of product filters can be found in the above documentation.
+
+query GetProductTypeInCollection($handle: String!, $value: String!) {
```

Comment:
@syfiawoo after toying with  the idea of `getProductProductTypeInCollection` and deciding it could be confusing, I'm going with your suggestion of `getProductsOfProductTypeInCollection`. The docs explain [productType](https://shopify.dev/api/storefront/2023-01/objects/Product#fields:~:text=A%20categorization%20that%20a%20product%20can%20be%20tagged%20with%2C%20commonly%20used%20for%20filtering%20and%20searching.) as _"a categorization that a product can be tagged with, commonly used for filtering and searching."_ - as it's a categorization I feel the new name fits the bill  😄